src/components/navigation/Navbar.scss

```
@@ -0,0 +1,103 @@
+@import "../../constants.scss";
+
+.navbar {
+  display: block;
+  position: fixed;
+  top: 20px;
+  left: 20px;
+  height:100%;
+  z-index: 1;
+  -webkit-user-select: none;
+  user-select: none;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: white;
+  transition: color 0.3s, underline 0.3s;
+}
+
+.navbar a:hover {
+  color: black;
+  text-decoration: underline;
+}
```

These can be brought inside the `.navbar` thanks to SCSS
```suggestion

  a {
    text-decoration: none;
    color: white;
    transition: color 0.3s, underline 0.3s;

    $:hover {
      color: black;
      text-decoration: underline;
    }
  }
}
```

---

Done - did you mean &:hover on 18?